### PR TITLE
Website: Redirect to latest bug reports thread

### PR DESCRIPTION
**Pull request by AnnikaCodes**

### CODE REVIEW

**Comment by DaWoblefet** [comment]
Probably should have /suggestions go to the correct subforum too while we're at it.

File: website/.htaccess [hunk 1]
```diff
@@ -89,7 +89,7 @@ RewriteRule ^help\/?$ https://play.pokemonshowdown.com/help [R=302,L]
 RewriteRule ^rooms?suggestions?\/?$ http://www.smogon.com/forums/threads/room-suggestions-are-no-longer-being-taken.3522130/ [R=302,L]
 RewriteRule ^suggestions?\/?$ http://www.smogon.com/forums/threads/suggestions.3534365/ [R=302,L]
```
Comment (Zarel) [inline website/.htaccess:90]:
```suggestion
RewriteRule ^suggestions?\/?$ https://www.smogon.com/forums/forums/suggestions.517/ [R=302,L]
```

**Comment by Zarel** [comment]
Other links should probably all be HTTPS while we're here.

**Comment by Zarel** [comment]
tyty

**Pull request merged**